### ISSUE TITLE
Event object with only 1 event raises AttributeError when loaded with the NeoMatlabIO

### ISSUE THREAD

**Issue by pietromarchesi** [NeuralEnsemble/python-neo#336]
For some reasons constructing an event object with only one event causes problems if the block is saved and loaded with the NeoMatlabIO. The following example raises `AttributeError: 'unicode' object has no attribute 'astype'`, while this problem does not occur when a second event is added (commented line). 

```python
import neo
from quantities import s

bl = neo.Block(name='Example Block')
seg = neo.Segment(name='segment 1')
ch = neo.ChannelIndex(name='Channel', index=0)
un  = neo.Unit(name='Unit')

train = neo.SpikeTrain(np.array([1,2,3]) * s,
                       t_start=0 * s,
                       t_stop=5 * s)
evt = neo.Event(times=np.array([0]) *s, labels=np.array(['Beginning'],dtype='S'))
#evt = neo.Event(times=np.array([0,1]) *s, labels=np.array(['Beginning','Middle'],dtype='S'))

seg.events.append(evt)
bl.segments.append(seg)
ch.units.append(un)
un.channel_index = ch
seg.spiketrains.append(train)

### SAVE AND READ THE BLOCK ###
from neo.io.neomatlabio import NeoMatlabIO
w = NeoMatlabIO(filename='exampleblock.mat')
w.write_block(bl)

r = neo.io.NeoMatlabIO(filename='exampleblock.mat')
rbl = r.read_block()
```

**Comment by johnpi** [comment]
I came across the same issue when trying to open a file that I converted from .smr to .mat using 

```python
from neo.io import Spike2IO, NeoMatlabIO
reader = Spike2IO(filename='filename.smr')
bl = reader.read(cascade=True, lazy=False)[0]
w = NeoMatlabIO(filename='filename.mat')
w.write_block(block)
```
Subsequently attempting to read from the file with

```python
reader = NeoMatlabIO(filename='filename.mat')
bl = reader.read_block(cascade=True, lazy=False)
```
The traceback I get is:

```python
Traceback (most recent call last):
...
...
    bl = reader.read_block(cascade=True, lazy=False)
  File "/.../neo/io/neomatlabio.py", line 234, in read_block
    bl_struct, 'Block', cascade=cascade, lazy=lazy)
  File "/.../neo/io/neomatlabio.py", line 367, in create_ob_from_struct
    cascade=cascade, lazy=lazy)
  File "/.../neo/io/neomatlabio.py", line 359, in create_ob_from_struct
    cascade=cascade, lazy=lazy)
  File "/.../neo/io/neomatlabio.py", line 398, in create_ob_from_struct
    item = item.astype(dt)
AttributeError: 'unicode' object has no attribute 'astype'

```

I then loaded filename.mat in Matlab and there were three event structures. The one of them had only one event with attributes:

```python
times_units : 's'
labels : '0'
times : 197.8415
```
As soon as I deleted this event structure and saved the block to the file using Matlab, the above code could successfully read the block. Therefore, it seems that even if there are other event structures in some cases an event structure with one event can cause the issue.

**Comment by zm711** [comment]
This has been fixed in the latest version of Neo (0.13.0)

```python
import neo
from quantities import s

bl = neo.Block(name='Example Block')
seg = neo.Segment(name='segment 1')

train = neo.SpikeTrain(np.array([1,2,3]) * s,
                       t_start=0 * s,
                       t_stop=5 * s)
evt = neo.Event(times=np.array([0]) *s, labels=np.array(['Beginning'],dtype='S'))
#evt = neo.Event(times=np.array([0,1]) *s, labels=np.array(['Beginning','Middle'],dtype='S'))

seg.events.append(evt)
bl.segments.append(seg)
seg.spiketrains.append(train)

### SAVE AND READ THE BLOCK ###
from neo.io.neomatlabio import NeoMatlabIO
w = NeoMatlabIO(filename='exampleblock.mat')
w.write_block(bl)

r = neo.io.NeoMatlabIO(filename='exampleblock.mat')
rbl = r.read_block()

```

Notice a couple of the older api components have been removed from the working example.
